modules/dreamview/conf/hmi_modes/mkz_standard_debug.pb.txt

```diff
@@ -32,7 +32,7 @@ cyber_modules {
 cyber_modules {
   key: "Localization"
   value: {
-    dag_files: "/apollo/modules/localization/dag/dag_streaming_msf_localization.dag"
+    dag_files: "/apollo/modules/localization/dag/dag_streaming_rtk_localization.dag"
```

Comment:
We may prefer MSF localization by default.

Reply:
LGSVL Simulator does not work with MSF localization for some reason. Since MSF is not open sourced, we need your help to figure out why it does not work. Before this is fixed, we have to use RTK for now.

Reply:
Ditto. We adapt to road test by default. Prefer adding dedicated mode for simulation purpose.

Reply:
Added a new mode `mkz_lgsvl.pb.txt`.